Okta.AspNet.Abstractions/Okta.AspNet.Abstractions.csproj

```diff
@@ -36,8 +36,8 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>okta.aspnet.public.snk</AssemblyOriginatorKeyFile>
     <DelaySign>true</DelaySign>
-    <AssemblyVersion>4.2.1.0</AssemblyVersion>
-    <FileVersion>4.2.1.0</FileVersion>
+    <AssemblyVersion>4.2.2.0</AssemblyVersion>
```

Comment:
This is a breaking change since we're dropping support for 4.5.x, so the updated version should be 5.0.0.0.
We should also update the readme and changelog indicating we no longer support 4.5.x but devs can still use 4.x.